src/components/sidebar/routePathView/NewRoutePathView.tsx

```diff
@@ -17,17 +55,17 @@ class NewRoutePathView extends React.Component{
             <ViewHeader
                 header='Luo uusi reitinsuunta'
             />
-            <div className={s.padding} />
-            <div className={s.padding} />
-            <RoutePathViewForm
-                isEditingDisabled={false}
-            />
-            <div>
-                <div className={s.sectionDivider} />
-                <div className={s.padding} />
+            <div className={s.section}>
```

Comment:
If you look at the HTML, there is this structure:

```
section
    div
          section
```
which doensn't seem right.

Also that 5 button section doesn't have a section wrapper

Reply:
Also note: before changing anything see what I proposed at .margin comment. I think there might be no need for this wrapper